packages/nextjs-mf/src/internal.ts

```diff
@@ -98,8 +98,13 @@ export const DEFAULT_SHARE_SCOPE_BROWSER: SharedObject = Object.entries(
 ).reduce((acc, item) => {
   const [key, value] = item as [string, SharedConfig];
 
-  acc[key] = { ...value, import: undefined };
+  acc[key] = { ...value, eager: undefined, import: undefined };
+
   // @ts-ignore
+  if (key === 'react' || key === 'react-dom') {
```

Comment:
outside the reduce functioni:
```suggestion
const REACT_DEPS = ['react', 'react-dom']
```
and here:
```suggestion
  if (REACT_DEPS.includes(key)) {
```